client/src/components/Nav/index.jsx

```diff
@@ -32,7 +51,10 @@ class AppNavBar extends Component {
           <Link to="/">
             <img id="logo" src="http://bit.ly/2beSCQg" alt="logo" />
           </Link>
-          <NavMenuIcon />
+          <NavMenuIcon
+            loggedIn={this.props.loggedIn}
```

Comment:
I remember making a comment that user prop can be used and having loggedIn prop is redundant